### PR TITLE
Email validation added

### LINKED ISSUE #142: Validation needs to be imporved in email fields.
**Actual Behaviour**
Validation in email fields is a bit inefficient, currently it only cheks the presence of `@` and text before and after it.

**Expected Behaviour**
Validation can improved, it should assure presence of `.` and extensions.

**Steps to reproduce it**
Open login page

**Would you like to work on the issue?**
Yes

GIF:-
ezgif com-video-to-gif 11
 It allows `1@2` as an email value

### PR DESCRIPTION
**Pull request by neeraj3029**
Fixes #142 

#### Changes proposed in this pull request:
Proper email validation is added to email field, now it warning is shown when proper email format is not followed.

#### Screenshots for the change: 
ezgif com-crop

### CODE REVIEW

**Comment by Ryan10145** [comment]
@neeraj3029 How exactly does your email validation check work? I know that there are some weird emails out there that contain multiple `.`s and I'm just making sure those emails would still be counted as valid. For instance, someone could have the email `first.last@xyz.abc.hs`

**Comment by neeraj3029** [comment]
@Ryan10145 correct me if I'm wrong, but I suppose the email format you have given in example is in a proper format. For example, `abc.xyz@iiitv.ac.in` , is a valid email.

**Comment by Ryan10145** [comment]
@neeraj3029 Ok, that’s good, LGTM :+1:

**Comment by neeraj3029** [comment]
@tstreamDOTh Please review!

**Comment by tstreamDOTh** [comment]
fix the conflicts

**Comment by neeraj3029** [comment]
@tstreamDOTh resolved!

File: src/settings.html [hunk 1]
```diff
@@ -14,6 +14,10 @@ <h3 class="hr-heading">Login</h3>
 				<hr />
 				<input type="email" id="username" placeholder="Email">
 				<br>
+      	<div id="emailWarning" style="display:none; color:red; margin-left: 10px">
```
Comment (samagragupta) [inline src/settings.html]: Please do the proper indentation.

Comment (neeraj3029) [reply src/settings.html]: @samagragupta I've changed the indentation, but somehow it looks different on browser. You can see the screenshot from the codebase, it looks fine. 
screenshot 2018-12-08 at 7 58 53 pm

**Comment by hkedia321** [comment]
@neeraj please fix conflicts.

**Comment by neeraj3029** [comment]
@hkedia321 @akshatnitd I've resolved the conflicts

**Pull request merged**